Code/Framework/AzCore/Tests/Math/AabbTests.cpp

```diff
@@ -555,8 +559,12 @@ namespace UnitTest
         const Vector3 expectedMin(929.289307f, -2.40615924e+38f, -10.0f);
         const Vector3 expectedMax(2.40615965e+38f, 2141.42139f, 10.0f);
 
-        EXPECT_THAT(aabbFromTransform.GetMin(), IsClose(expectedMin));
-        EXPECT_THAT(aabbFromTransform.GetMax(), IsClose(expectedMax));
+        EXPECT_NEAR(aabbFromTransform.GetMin().GetX(), expectedMin.GetX(), AZ::Constants::Tolerance);
+        EXPECT_TRUE(IsCloseMag(aabbFromTransform.GetMin().GetY(), expectedMin.GetY(), AZ::Constants::Tolerance));
+        EXPECT_NEAR(aabbFromTransform.GetMin().GetZ(), expectedMin.GetZ(), AZ::Constants::Tolerance);
+        EXPECT_TRUE(IsCloseMag(aabbFromTransform.GetMax().GetX(), expectedMax.GetX(), AZ::Constants::Tolerance));
+        EXPECT_NEAR(aabbFromTransform.GetMax().GetY(), expectedMax.GetY(), AZ::Constants::Tolerance);
+        EXPECT_NEAR(aabbFromTransform.GetMax().GetZ(), expectedMax.GetZ(), AZ::Constants::Tolerance);
```

Comment:
question: Just curious why these got expanded out? There is an `IsCloseTolerance` that could be used to keep things more concise potentially?

Reply:
This needed to be expanded to use `IsCloseMag` when comparing the 2 elements that are meant to be equal to 1e+38, which are huge number and the regular `IsCloseTolerance` cannot handle.

I tried to add the `IsCloseMag` matcher, but unfortunately Vector3 doesn't have the member function `IsCloseMag`, so that would be needed to be added, but then you'd want it in Vector2, Vector4, Matrix4, add unit tests, etc. which is too much of a change to get mixed up with this PR. So for now I just checked element by element using `IsCloseMag` when needed and in the future `IsCloseMag` matcher could be added.

Reply:
Ah I see understood, thanks for clarifying 👍